### PR TITLE
Update metadata

### PR DESCRIPTION
**Pull request by Billito00**
Updating and adding links where they where missing or outdated.

### CODE REVIEW

**Comment by blanky0230** [comment]
Thank you for your contributions! Would you be so kind to give as a borad overview what you've changed? (add a description).

**Comment by n-se** [comment]
On a merge all commits are automatically squashed by default to one single commit. It is not really needed to do a manual squash before the merge.

File: tutorials/install-lamp-on-ubuntu-debian/01.en.md [hunk 1]
```diff
@@ -9,7 +9,10 @@ tags: ["MySQL", "PHP", "Apache", "LAMP", "Lang:Bash"]
 author: "phz1"
 author_link: "https://github.com/phz1"
 author_img: "https://avatars3.githubusercontent.com/u/12565299"
+language: "en"
+available_languages: ["en"]
 header_img: "howto-install-lamp-stack.jpg"
+
```
Comment (n-se) [inline tutorials/install-lamp-on-ubuntu-debian/01.en.md]: Is the new line needed here? I mark it in this review as I don't know if maybe an issue occurs with a new line here.

**Comment by fdrechsler** [comment]
Thank you Sebastian!

**Pull request merged**